modules/terraform/cmd.go

```diff
@@ -48,14 +59,7 @@ func RunTerraformCommand(t testing.TestingT, additionalOptions *Options, args ..
 func RunTerraformCommandE(t testing.TestingT, additionalOptions *Options, additionalArgs ...string) (string, error) {
 	options, args := GetCommonOptions(additionalOptions, additionalArgs...)
 
-	cmd := shell.Command{
-		Command:           options.TerraformBinary,
-		Args:              args,
-		WorkingDir:        options.TerraformDir,
-		Env:               options.EnvVars,
-		OutputMaxLineSize: options.OutputMaxLineSize,
-	}
-
+	cmd := generateCommand(additionalOptions, additionalArgs...)
```

Comment:
Build failed because this should actually be:
```suggestion
	cmd := generateCommand(options, args...)
```
This needs to be fixed with the other functions as well.